src/main/resources/liquibase.properties

```diff
@@ -0,0 +1,4 @@
+url=jdbc:mysql://localhost:3306/bookstore
+username=${DATABASE_USER}
+password=${DATABASE_PASSWORD}
+changelog-file=db/changelog/db.changelog-master.yaml
```

Comment:
also, not sure this file is actually necessary, `application.properties` should be enough